### DISCUSSION TITLE
Download Helper not recognizing latest companion helper in Ubuntu 22.04LTS and Firefox ESR

### DISCUSSION THREAD

**Discussion by JaimeACruz** [aclap-dev/video-downloadhelper#40] [Help and Q&A] [answered]
### Your question

I have Firefox-ESR 115.7.0 installed from deb (NOT snap or flatpak). I've installed the vdhcoapp-2.0.10-linux-x86_64.deb successfully. I've tried restarting Firefox, logging off and back on and even rebooting but VDH Version 8.1.4.0 REFUSES to recognize that the app is installed.  HELP! It worked perfectly until the latest update. All searches I've done on this problem point to Snap or Flatpak but I am DEFINITELY using neither; I definitely have the version from the ppa:mozillateam/ppa repository.

### Addon & CoApp version and details

Firefox 115.7.0ESR, Ubuntu 22.04.3LTS, vdhcoapp-2.0.10-linux-x86_64.deb

**Comment by paulrouget** [comment]
What's the output of:

```
/opt/vdhcoapp/vdhcoapp --info
```

and

```
/opt/vdhcoapp/vdhcoapp install
```

?

**Reply by JaimeACruz** [reply]
/opt/vdhcoapp/vdhcoapp --info
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "/opt/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/jaimec",
  "converterBinary": "/opt/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

/opt/vdhcoapp/vdhcoapp install
Installing…
Flatpak is installed. Making the coapp available from browser sandboxes:
Linked coapp within org.mozilla.firefox.
Linked coapp within com.brave.Browser.
Linked coapp within com.google.Chrome.
Linked coapp within com.google.ChromeDev.
Linked coapp within org.chromium.Chromium.
Linked coapp within com.github.Eloston.UngoogledChromium.
Linked coapp within com.microsoft.Edge.
Linked coapp within com.microsoft.EdgeDev.
Writing /home/jaimec/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json
Writing /home/jaimec/.var/app/org.mozilla.firefox/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json
Writing /home/jaimec/.config/google-chrome/NativeMessagingHosts/net.downloadhelper.coapp.json
VdhCoApp : VdhCoApp is ready to be used

**Comment by JaimeACruz** [comment, marked as answer]
That second command seems to have resolved the issue as now the add-on recognizes the helper app and it's working properly.  Thank you!
